test_radix.py

```diff
@@ -0,0 +1,58 @@
+from radix_sort import radixsort
+from random import shuffle
+import pytest
+
+
+def test_randoms():
+    x = range(100)
+    randoms = x[:]
+    shuffle(randoms)
+    radixsort(randoms)
+    y = randoms
+    assert x == y
+
+
+def test_empty():
+    x = []
+    y = x
+    radixsort(x)
+    assert x == y
+
+
+def test_reversed():
+    x = []
+    for i in range(1, 10):
+        x.append(i)
+    y = []
+    for i in reversed(x):
+        y.append(i)
+    radixsort(y)
+    z = y
+    assert z == x
+
+
+def test_ordered():
+    x = []
+    for i in range(1, 10):
+        x.append(i)
+    radixsort(x)
+    z = x
+    assert z == x
```

Comment:
This test is meaningless.  You point both the `x` and the `z` variables at exactly the same value and then assert that they are equal.  Means nothing.